data/constants/keycodes/keycodes_0.0.1.hjson

```diff
@@ -0,0 +1,94 @@
+{
+    "ranges": {
+        "0x0000/0x00FF": {
+            "define": "QK_BASIC"
+        },
+        "0x0100/0x1EFF": {
+            "define": "QK_MODS"
+        },
+        "0x2000/0x1FFF": {
+            "define": "QK_MOD_TAP"
+        },
+        "0x4000/0x0FFF": {
+            "define": "QK_LAYER_TAP"
+        },
+        "0x5000/0x001F": {
+            "define": "QK_TO"
+        },
+        "0x5020/0x001F": {
+            "define": "QK_MOMENTARY"
+        },
+        "0x5040/0x001F": {
+            "define": "QK_DEF_LAYER"
+        },
+        "0x5060/0x001F": {
+            "define": "QK_TOGGLE_LAYER"
+        },
+        "0x5080/0x001F": {
+            "define": "QK_ONE_SHOT_LAYER"
+        },
+        "0x50A0/0x001F": {
+            "define": "QK_LAYER_TAP_TOGGLE"
+        },
+        // 0x50C0/0x001F - UNUSED
+        // 0x50E0/0x001F - UNUSED
+        "0x5100/0x00FF": {
+            "define": "QK_ONE_SHOT_MOD"
+        },
+        "0x5200/0x01FF": {
+            "define": "QK_LAYER_MOD"
+        },
+        "0x5400/0x00FF": {
+            "define": "QK_SWAP_HANDS"
+        },
```

Comment:
Do we want to reserve a second range for things like `SH_TG`, `SH_TT`, …, which are currently implemented as if they were `SH_T(x)` with some special values in place of `x` (`OP_SH_TOGGLE`, `OP_SH_TAP_TOGGLE`, …) where those special values start from `0xF0`, overlaying a part of the basic keycode range?  Or do we just declare that a part of the basic keycode range would not be fully usable?

#16076 moved the mouse keycodes because of a similar problem inside the `LT()` implementation, but in that case the problem is in the `action_t` encoding, which should hopefully go away at some time, and the keycodes for the problematic features are distinct; but the special keycodes for the swap hands feature are actually overlapping the `SH_T(x)` range.

Or maybe do a single range for `QK_SWAP_HANDS` which extends past `0x00FF` (and then use the leftover space to pack some of the `0x001F` blocks).

Reply:
Till the action issues are resolved, its probably safe to assume `0xF0-0xFF` isnt a safe range to use.

Moving the various "static" keycodes to somewhere else could be okay.